libp2p/Cargo.toml

```diff
@@ -65,6 +66,7 @@ json = ["libp2p-request-response?/json"]
 kad = ["dep:libp2p-kad", "libp2p-metrics?/kad"]
 macros = ["libp2p-swarm/macros"]
 mdns = ["dep:libp2p-mdns"]
+memory-connection-limits = ["dep:libp2p-memory-connection-limits"]
```

Comment:
Thanks for adding the feature-flag! We are adding some dependencies here so I think this is worth having. (Note that the previous two connection limits crates are not feature-flagged)